package.json

```diff
@@ -9,6 +9,8 @@
     "emulator": "firebase ext:dev:emulators:start --test-params=test-params.env --import=dataset --project=firebase-extension-test",
     "lint": "npm run lint --prefix functions",
     "build": "npm run build --prefix functions",
-    "install:functions": "npm install --prefix functions"
+    "install:functions": "npm install --prefix functions",
+    "test": "npm run test --prefix functions",
+    "test-coverage": "npm run test-coverage --prefix functions"
```

Comment:
When using `--prefix` flag the user must have `npm >= v7`. It should be added in the CONTRIBUTING.md